autosklearn/evaluation/__init__.py

```diff
@@ -324,7 +349,12 @@ def run(
                 'traceback': exception_traceback,
                 'error': error_message
             }
-            return StatusType.CRASHED, self.cost_for_crash, 0.0, additional_info
+            return StatusType.CRASHED, self.worst_possible_result, 0.0, cast(
```

Comment:
Do you think it would be more readable if you'd define the dictionary `additional_info` upfront?

Reply:
thanks, I have done so